internal/controller/setup.go

```diff
@@ -142,7 +142,7 @@ func (r *WatcherReconciler) SetupWithManager(mgr ctrl.Manager, options ctrlrunti
 		return errRestConfigIsNotSet
 	}
 	var err error
-	r.IstioClient, err = istio.NewVersionedIstioClient(r.RestConfig, r.EventRecorder,
+	r.IstioClient, err = istio.NewIstioClient(r.RestConfig, r.EventRecorder,
```

Comment:
Removed `versioned` as the constructor doesn't include any special semantics with regards to version.